docs/interpreter/ignite.md

```diff
@@ -89,7 +89,7 @@ After configuring Ignite interpreter, create your own notebook. Then you can bin
 
 ![Binding Interpreters](../assets/themes/zeppelin/img/docs-img/ignite-interpreter-binding.png)
 
-For more interpreter binding information see [here](http://zeppelin.apache.org/docs/manual/interpreters.html).
+For more interpreter binding information see [here](../manual/dynamicform.html).
```

Comment:
You need to change this link `../manual/dynamicform.html` to `../manual/interpreters.html#what-is-interpreter-setting` :)